codes/typescript/chapter_hashing/array_hash_map.ts

```diff
@@ -0,0 +1,139 @@
+/*
+ * File: array_hash_map.ts
+ * Created Time: 2022-12-20
+ * Author: Daniel (better.sunjian@gmail.com)
+ */
+
+/* 键值对 Number -> String */
+class Entry {
+  public key: number;
```

Comment:
The indentation is 4 spaces for this project. Please compare the code with the previous TS code and match the formmating.